File: src/main/java/com/back/domain/member/memberGame/dto/MemberGameAddRequest.java
package com.back.domain.member.memberGame.dto;
import com.back.domain.member.memberGame.StatusEnum;
import jakarta.validation.constraints.Min;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;

public record MemberGameAddRequest(
        @NotBlank
        String platform,
        @Min(0)
        double playtime,
        boolean isFavorite,
        @NotNull
        StatusEnum status,
        int gameId
) {
}

File: src/main/kotlin/com/back/domain/member/memberGame/dto/MemberGameAddRequest.kt
package com.back.domain.member.memberGame.dto

import com.back.domain.member.memberGame.StatusEnum
import jakarta.validation.constraints.DecimalMin
import jakarta.validation.constraints.NotBlank

data class MemberGameAddRequest(
    @field:NotBlank val platform: String,
    @field:DecimalMin("0.0") val playtime: Double,
    val isFavorite: Boolean,
    val status: StatusEnum?,
    val gameId: Long,
)
